app/models/organisation.rb

```diff
@@ -5,7 +5,7 @@ class Organisation < ApplicationRecord
 
   has_and_belongs_to_many :providers
 
-  validates :name, presence: true
+  validates :name, presence: { message: "Enter organisation name" }
```

Comment:
Can we add this to the translations under an appropriate heading, like the others?

Reply:
Done